res/sections/68-EsLes14032017.tex

```diff
@@ -114,25 +114,158 @@ \chapter{Business Continuity and Disaster Recovery}
   title={Domanda su BCP},
   label={bcdr5}
  ]
- 
- \Question Durante un audit del piano di continuità del business, la scoperta 
+
+ \Question Durante un audit del piano di continuità del business, la scoperta
  più sconcertante sarebbe:
  \begin{enumerate}
-   \item L'albero delle chiamate non è stato controllato in maniera debita 
+   \item L'albero delle chiamate non è stato controllato in maniera debita
    negli ultimi sei mesi
    \item L'analisi sull'impatto del business non è stata aggiornata quest'anno
-   \item Un test del sistema di backup e recupero non è stato eseguito 
+   \item Un test del sistema di backup e recupero non è stato eseguito
    regolarmente
    \item Il sito di backup manca di un UPS
  \end{enumerate}
-  
+
 \end{Exercise}
 
 \begin{Answer} [
   ref={bcdr5},
   number={5}
  ]
- 
+
  \Question Risposta esatta: 3
- 
+
+\end{Answer}
+
+% Altro esercizio
+
+\begin{Exercise} [
+  title={Quiz},
+  label={bcdr6}
+ ]
+
+ \Question Il primo e più importante test per verificare il business
+ continuity plan è:
+ \begin{enumerate}
+   \item \textit{Fully operational test}
+   \item \textit{Preparedness Test}
+   \item \textit{Security Test}
+   \item \textit{Desk-Based paper test}
+ \end{enumerate}
+
+\end{Exercise}
+
+\begin{Answer} [
+  ref={bcdr6},
+  number={6}
+ ]
+
+ \Question Risposta esatta: ?
+
+\end{Answer}
+
+% Altro esercizio
+
+\begin{Exercise} [
+  title={Quiz},
+  label={bcdr7}
+ ]
+
+ \Question Quando accade un disastro, qual è la massima priorità:
+ \begin{enumerate}
+   \item Assicurarsi che tutte le persone siano in salvo
+   \item Minimizzare la perdita dei dati salvando i dati importanti
+   \item Recuperare i nastri di backup
+   \item Chiamare un \textit{manager}
+ \end{enumerate}
+
+\end{Exercise}
+
+\begin{Answer} [
+  ref={bcdr7},
+  number={7}
+ ]
+
+ \Question Risposta esatta: 1
+
+\end{Answer}
+
+% Altro esercizio
+
+\begin{Exercise} [
+  title={Quiz},
+  label={bcdr8}
+ ]
+
+ \Question Un processo documentato dove si determina l'operazione
+ IT \emph{più} cruciale dalla prospettiva del business.
+ \begin{enumerate}
+   \item \textit{Business Continuity Plan}
+   \item \textit{Disaster Recovery Plan}
+   \item \textit{Restoration Plan}
+   \item \textit{Business Impact Analysis}
+ \end{enumerate}
+
+\end{Exercise}
+
+\begin{Answer} [
+  ref={bcdr8},
+  number={8}
+ ]
+
+ \Question Risposta esatta: 4
+\end{Answer}
+
+
+
+% Altro esercizio
+
+\begin{Exercise} [
+  title={Quiz},
+  label={bcdr9}
+ ]
+
+ \Question L'obiettivo principale di un Post-Test è:.
+ \begin{enumerate}
+   \item Scrivere un report a scopo di audit
+   \item Ritornare al servizio normale
+   \item Valutare l'efficacia del test e aggiornare il \textit{response
+   plan}
+   \item Fare un resoconto sul test al management
+ \end{enumerate}
+
+\end{Exercise}
+
+\begin{Answer} [
+  ref={bcdr9},
+  number={9}
+ ]
+
+ \Question Risposta esatta: ?
```

Comment:
Primary goal of post test is to evaluate test (e poi fare report al management!!)